charts/datadog-crds/README.md

```diff
@@ -0,0 +1,28 @@
+# Datadog CRDs
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1](https://img.shields.io/badge/AppVersion-1-informational?style=flat-square)
+
+This chart was designed to allow other "datadog" charts to share `CustomResourcesDefinitions` such as the `DatadogMetric`.
+
+## How to use Datadog Helm repository
+
+You need to add this repository to your Helm repositories:
+
+```
+helm repo add datadog https://helm.datadoghq.com
+helm repo update
+```
+
+## Prerequisites
+
+This chart can be used with Kubernetes `1.11+` or OpenShift `3.11+` to the requirement to support `CustomResourcesDefinitions`.
```

Comment:
```suggestion
This chart can be used with Kubernetes `1.11+` or OpenShift `3.11+` since  `CustomResourceDefinitions` are supported starting with these versions.
```